easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -83,6 +95,10 @@ def configure_step(self, srcdir=None, builddir=None):
                 srcdir = default_srcdir
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
+
+        if '-DCMAKE_BUILD_TYPE' not in self.cfg['configopts']:
+            options.append("-DCMAKE_BUILD_TYPE=%s" % self.cfg['buildtype'])
```

Comment:
we should honor backward compatibility, and not blindly add configure options that (may) affect the output of a build, and especially not generate confusing configure commands where two different values are specified for the same option, making it unclear which wins (last wins is a decent assumption, but it's not always that way, so people may wonder, etc.)